src/bentoml/_internal/server/runner_app.py

```diff
@@ -244,18 +248,37 @@ async def infer_batch(
 
             infer = self.dispatchers[runner_method.name](infer_batch)
         else:
+            if runner_method.config.is_stream:
```

Comment:
`elif`?

Reply:
thought that nesting this would make sense, i.e the first part is for batching, second part is for non batch.

Reply:
Ah, I see. I think nested ifs can be a little hard to read, but if you feel like this distinction is important, I also feel like it makes more sense for the top-level difference to be streamed / not streamed first, then batched / unbatched.